src/main/kotlin/com/dasiusp/cleber/pdf/HelloWorldPdfGenerator.kt

```diff
@@ -1,34 +1,55 @@
 package com.dasiusp.cleber.pdf
 
-import com.itextpdf.text.Document
-import com.itextpdf.text.Phrase
+import com.dasiusp.cleber.pdf.PDFFont.bodyFont
+import com.dasiusp.cleber.pdf.PDFFont.bottomFont
+import com.dasiusp.cleber.pdf.PDFFont.titleFont
+import com.dasiusp.cleber.pdf.PDFFont.tokenFont
+import com.itextpdf.text.*
 import com.itextpdf.text.pdf.PdfWriter
 import java.io.File
 import java.io.FileOutputStream
+import java.time.LocalDate
 
+data class Certificate(
+    val personName: String,
+    val date: LocalDate,
+    val eventType: String,
+    val eventName: String,
+    val duration: Int,
+    val token: String
+)
 class HelloWorldPdfGenerator(
     private val outputDirectory: String
 ) {
-    
+
     init {
         createOutputDirectory()
     }
 
     private fun createOutputDirectory() {
         File(outputDirectory).mkdirs()
     }
-    
-    fun createPdf(name: String) {
-        val targetFile = File(outputDirectory, "$name.pdf")
-        writeDocument(targetFile, name)
+
+    fun createPdf(certificate: Certificate) {
+        val targetFile = File(outputDirectory, "${certificate.token}.pdf")
+        writeDocument(targetFile, certificate)
     }
-    
-    private fun writeDocument(targetFile: File, name: String) {
+
+    private fun writeParagraph(text: String, font: Font, alignment: Int): Paragraph {
+        val paragraph = Paragraph(text,font)
+        paragraph.alignment = alignment
+        return paragraph
+    }
+
+    private fun writeDocument(targetFile: File, certificate: Certificate) {
         val document = Document()
         PdfWriter.getInstance(document, FileOutputStream(targetFile))
         
         document.use {
-            add(Phrase("Hello, $name!"))
+            add(writeParagraph("Certificado de Participação", titleFont, 1))
```

Comment:
Todas essas linhas possuem um argumento de alinhamento. Precisamos retirar esse [número mágico](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad) daqui.

Para isso, podemos ver vários campos com boa nomenclatura na [documentação da classe `Element`](https://api.itextpdf.com/iText5/5.5.13/). Dê uma olhada nos `Align.XXX`. A IDE deve te ajudar a completar esse cara. Ficaria algo tipo:

```
writeParagraph("Foo", titleFont, Element.ALIGN_XXX)
```

Além disso, quero extrair cada comportamento único daqui para um método à parte.

- `addTitle()`
- `addBody()`
- `addPlaceAndDate()`
- `addToken()`